README.md

```diff
@@ -95,11 +92,11 @@ data class FingerprintAndroidConfiguration(
 )
 ```
 
-Add your configuration while initializaion.
+Add your configuration during the initializaion:
 
 ```kotlin
 val fingerprintAndroidAgent =
-            FingerprintAndroidAgentFactory.getInitializedInstance(applicationContext, configuration)
```

Comment:
why do we need to have `FingerprintAndroid` in every name? Can we namespace everythign to keep things simpler?